### DISCUSSION TITLE
Model doesn't open in Safari on mac or iOS

### DISCUSSION THREAD

**Discussion by apecollector** [google/model-viewer#3365] [Q&A]
Works in Chrome on mac but not Safari, also doesn't work on iOS 15. 

Web demo: https://modz-viewer.vercel.app/

The model is here: https://modz-viewer.vercel.app/brodus.gltf, also opens fine in Blender.

Code is nothing special, just using:

```
    <model-viewer
      bounds="tight"
      src="/brodus.gltf"
      ar
      ar-modes="scene-viewer webxr"
      camera-controls
      environment-image="neutral"
      poster="/poster.webp"
      shadow-intensity="1"
      seamless-poster
      enable-pan
      enable-controls
    />
```

**Comment by samaneh-kazemi** [comment]
Hi @apecollector, thanks for reaching out. I just looked at your website source through devtools. Can you try passing 'quick-look' to 'ar-modes' and try again? 
```
ar-modes="webxr scene-viewer quick-look"
```

**Comment by apecollector** [comment]
Hi @samaneh-kazemi, I had that in prior but removed it thinking it might be the cause based on another issue I saw.

Here's a glitch link: https://rowan-viridian-liver.glitch.me/

I understand it could be tricky to test and reproduce since it's Mac and iOS specific.

**Reply by elalish** [reply]
@apecollector Can you be more specific than "doesn't work"? Are you talking about 3D, AR? A screen recording would be great, of both the working and non-working cases. One thing: are you sure you meant to use absolute paths? I don't usually see things starting with `/`.

**Reply by apecollector** [reply]
Hi, the 3D model doesn't load at all, it's stuck in the "loading" state. I'm sure I meant to use absolute path, like I said it works on desktop but not iOS, not really sure how to do a screen recording on iOS.
